Lib/timeit.py

```diff
@@ -326,7 +335,7 @@ def callback(number, time_taken):
         try:
             number, _ = t.autorange(callback)
         except:
-            t.print_exc()
+            t.print_exc(colorize=colorize)
```

Comment:
Shouldn't `colorize` be configured in the constructor of `Timer` instead of every usage of `t.print_exc`? 

This creates issues when using `Timer` programmatically. 

Also, this will create a friction point with any future requests that need the output of `Timer` to be colorized.

Reply:
Because different `file`s can be specified for the same Timer object and we don't want this logic to apply unconditionally.